.github/workflows/build.yml

```diff
@@ -86,6 +86,7 @@ jobs:
         pip install .
         pip install .[testing]
         pip install -r docs/requirements.txt
+        pip install .
```

Comment:
why are there now 2 pip install . lines?

Reply:
Sorry I added this as I was trying to fix CI. Removed.